packages/compass/src/main/menu.spec.ts

```diff
@@ -111,6 +143,152 @@ describe('CompassMenu', function () {
       }).not.to.throw();
     });
 
+    describe('manual update menu item', function () {
+      const bw = new BrowserWindow({ show: false });
+      beforeEach(() => {
+        App.emit('new-window', bw);
+      });
+      describe('darwin', () => {
+        beforeEach(function () {
+          if (process.platform !== 'darwin') this.skip();
```

Comment:
`setApplicationMenu` looks at the value of `process.platform` to determine how to set the menu, so I restricted these tests to only run on their respective platforms.

https://github.com/electron/electron/blob/38850112200a8fa5e17c7c393b409bb33a747f9a/lib/browser/api/menu.ts#L168